main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -2387,6 +2389,20 @@ object Parser2 {
       close(mark, TreeKind.Expr.Do)
     }
 
+    private def invokeMethod2Expr()(implicit s: State): Mark.Closed = {
+      assert(at(TokenKind.NameLowerCase) || at(TokenKind.NameUpperCase))
+      val mark = open()
+      name(Set(TokenKind.NameUpperCase, TokenKind.NameLowerCase), context = SyntacticContext.Expr.OtherExpr)
+      // TODO: emit an error if we are not at an hash here
```

Comment:
Something we have done with some other projects (especially ones that tough a lot of phases) is to have a "tag" to put on all the TODOs. This makes it easier to track them all since we can just search the project. For example, we have `TODO ASSOC-TYPES blah blah blah`.

I propose you pick a reasonable tag for your project (e.g. INTEROP or an abbreviation thereof) and tag your TODOs with it.